backend/fleet_management/api.py

```diff
@@ -1,11 +1,25 @@
-from rest_framework import views
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import generics, filters, views
 from rest_framework.response import Response
-
-from .serializers import UserSerializer
+from .serializers import UserSerializer, ProjectSerializer
+from .models import Project
+from rest_framework.permissions import IsAuthenticated
```

Comment:
Please keep imports sorted and grouped by:
* Python builtins
* 3rd party libraries (like django, rest_framework)
* our modules